.github/workflows/test.yml

```diff
@@ -39,6 +39,15 @@ jobs:
 
     - name: Check Formatting
       run: cargo +nightly fmt --all -- --check
+    
+    - name: Check Stable Compilation
+      run: cargo build --all-features
+
+    - name: Check Nightly Compilation
+      run: cargo +nightly build --all-features
+    
+    - name: Check Bench Compilation
+      run: cargo +nightly bench --no-run --profile=dev --all-features
```

Comment:
Will we look at the result? It won't give a diff of the last run will it?
It seems too much for a PR especially.

Reply:
Note the `--no-run` argument. This will not execute the benchmarks, just check that they compile (and even use the `dev` profile to get fast build time instead of fast builds).

Reply:
We broke the benchmarks compilation previously, since they are not covered in the CI. This did break as a consequence https://pseitz.github.io/tantivy_bench_graph/index.html

Reply:
ah got it!